stdlib/contextlib.pyi

```diff
@@ -137,6 +137,9 @@ elif sys.version_info >= (3, 7):
         func: Callable[..., AsyncGenerator[_T_co, Any]]
         args: tuple[Any, ...]
         kwds: dict[str, Any]
+        async def __aexit__(
+            self, typ: type[BaseException] | None, value: BaseException | None, traceback: TracebackType | None
+        ) -> bool | None: ...
```

Comment:
It actually always returns bool (on main at least), but I don't think that really matters. If we change this we have to change it above on line 131 too.